app/controllers/idv/usps_controller.rb

```diff
@@ -1,11 +1,13 @@
+# :reek:TooManyMethods
```

Comment:
This controller has gotten to almost 160 lines, which to me looks like a signal that some of this should be moved into a service